### ISSUE TITLE
Find in selection button doesn't have blue border when selected

### ISSUE THREAD

**Issue by rebornix** [Microsoft/vscode#25751]
- VSCode Version: 1.12-insiders
- OS Version: macOS

Steps to Reproduce:

1. Select some text, cmd+f
2. Click on Find in selection button, it has a grey background but no borders
3. Press tab to cycle through all buttons in the UI, you can't notice when Find in selection is selected.

**Comment by rebornix** [comment]
Dup of #25470.
